### PR TITLE
FPW-42 Hide menu for small devices

### PR DESCRIPTION
**Pull request by kmkr**
For small devices:

* Hamburger icon is removed and replaced with a text button "Menu". The :hamburger: is gone.
* Click on menu button shows the menu
* Clicking a link in the menu opens the link and hides the menu
* Clicking elsewhere on the page hides the menu

For large devices:

* Menu is always shown (as before)
* Menu icon is not shown (as before)

cc @sveisvei @Skadefryd

### CODE REVIEW

**Comment by runeh** [comment]
Afaik, there have been tests showing that the word "menu" converts better than the icon I think, so maybe that should be the default?

File: docs/docs.js [hunk 1]
```diff
@@ -1,39 +1,43 @@
...
+        }
+    });
+}
 
-const treeMenu = document.getElementById('tree-menu');
 if (treeMenu) {
     treeMenu.addEventListener('click', function (e) {
         if (e.target) {
-            const ref = e.target.getAttribute('data-ref');
+            var ref = e.target.getAttribute('data-ref');
```
Comment (kmkr) [inline docs/docs.js]: Changed this since we're not Babelifying the script

File: docs/docs.js [hunk 2]
```diff
@@ -1,39 +1,43 @@
-// // Menu button element
-// var menu = document.getElementById('menu');
+var menuButton = document.getElementById('menu');
```
Comment (runeh) [inline docs/docs.js]: There is, or at least used to be, a rule of thumb that IDs and classes are not used for scripting, thus in mfinn at least, we'd use "data-button-menu" attribute or similar to select the elements.

File: docs/docs.js [hunk 3]
```diff
@@ -1,39 +1,43 @@
...
-//         classString += ' ' + className;
-//     }
-//     else {
-//         classString = classString.substr(0, nameIndex) + classString.substr(nameIndex+className.length);
-//     }
-//     element.className = classString;
-// }
+function toggleClass (elem, clazz) {
+    if (hasClass(elem, clazz)) {
+        elem.className = elem.className.replace(clazz, '');
```
Comment (runeh) [inline docs/docs.js]: This comes back to the polyfill question perhaps? It would be nice to use ele.classList ( http://caniuse.com/#feat=classlist )

**Comment by kmkr** [comment]
> Afaik, there have been tests showing that the word "menu" converts better than the icon I think, so maybe that should be the default?

Agreed with @Skadefryd to remove the burger in favor of "Menu". Fixed and pushed.

File: docs/docs.js (hunk 3 above)
Comment (kmkr) [inline docs/docs.js]: Hmm, yeah, I guess. It's no problem for me to include polyfills, but it doesn't seem to be this PR's responsibility to do so. I can do it if you disagree, or I could do it in a separate PR and update code to use improved APIs.

Comment (runeh) [inline docs/docs.js]: Mm. For something this tiny and self-contained, I guess it doesn't matter much to use the "old school" way anyway.

**Pull request merged**